src/RTCPeerConnection.ts

```diff
@@ -631,6 +633,10 @@ export default class RTCPeerConnection extends defineCustomEventTarget(...PEER_C
                 track
             ] = this.getReceivers().map(r => r.track).filter(t => t?.id === ev.trackId);
 
+            if (track?.readyState.toLowerCase() === 'ended') {
```

Comment:
No need to lower case it, it's always lower case.

Reply:
But in my case, the readyState's 1st character is always 'E'
{
    "readyState": "Ended",
    "id": "1b7cfbb7-55a1-4cb5-8873-17338327e7ce",
    "peerConnectionId": 0,
    "enabled": true,
    "kind": "video",
    "remote": true
}

Reply:
This was fixed in master earlier in the week.

Reply:
It's cool! :smiley: